### DISCUSSION TITLE
Option to autodelete content an user has both requested and watched

### DISCUSSION THREAD

**Discussion by stpnwf** [seerr-team/seerr#252] [Ideas]
I would like that for some users of my choosing, all the content **that they have requested** be deleted X days **after they have watched it**. That way I only keep in my library what I find worth keeping, and dont waste storage space on something that will only be watched once.

The users I have given access to Jellyseer are friends of mine, and they only use my Jellyfin instance as a streaming option to watch content mostly once. I trust them enough to enable auto approve, but I don't wanna store all that content forever (unlike the content I download).

My suggestion is for that feature to be added as an option in users settings where we can choose after how many days content those users request will be deleted once they watch it.

**Comment by SubatomicCat** [comment]
Yes, this would be very useful! I have a similar problem. I don't want to store the content some people download forever. Once they watch it then I don't mind if it is deleted. I would love to save up on my storage space like that.

I would also like that content be deleted if it goes longer than 5 or 7 days without watching for the accounts I enable this on. I too turn on auto approve, but sometimes people download a bunch of movies/shows and forget to watch them. 

These features would be much welcome imo.

**Reply by stpnwf** [reply]
Yes, being able to delete things people don't watch would be really nice too!

**Comment by NoobOnTour024** [comment]
Same here. There should be some sort of an auto-delete mechanism.

**Comment by AintGotNoLoveToday** [comment]
perhaps this is helpfull. In case you use jellyfin as media-server:
https://github.com/shemanaev/jellyfin-plugin-media-cleaner

**Comment by kevincox** [comment]
I would love to see some form of full lifecycle management. I had a different approach that I opened a discussion for here: https://github.com/seerr-team/seerr/discussions/2324

I don't think I would like it to be fully automatic. Also I don't think this solution handles multiple users watching the same content well.

For now I am telling people to use the "Report" functionality to signal that they are done with some content. But that leaves a lot to be desired.
